subgraph-js/src/printSubgraphSchema.ts

```diff
@@ -131,6 +135,10 @@ function isSchemaOfCommonNames(schema: GraphQLSchema): boolean {
     return false;
   }
 
+  if (schema.astNode?.description) {
+    return false
+  }
```

Comment:
I don't understand why this change is needed either.

Reply:
Similar to above in that it's not needed if you either have schema-definition directives (e.g. `@contact`) or if you define types for `mutation` or `subscription`, or a type for `query` other than `Query`. As before, removing this line will cause the test I modified to break.

Reply:
Got it. So this entire file is as much of a fork of `graphql-js`'s `printSchema` as possible. Let's follow what they've done and handle this check outside of this function. That decision is in line with my feeling that this doesn't _quite_ belong in this function based on its name.

See here, the description is inspected before even calling `isSchemaOfCommonNames`. I'd like it if we did the same.
https://github.com/graphql/graphql-js/blob/47bd8c8897c72d3efc17ecb1599a95cee6bac5e8/src/utilities/printSchema.ts#L73